.env.development

```diff
@@ -1,10 +0,0 @@
-NEXT_PUBLIC_API=https://dev-k8s.treetracker.org/query
-NEXT_PUBLIC_TILE_SERVER_URL=https://{s}.treetracker.org/tiles/
-NEXT_PUBLIC_TILE_SERVER_SUBDOMAINS=dev-k8s
-NEXT_PUBLIC_TILE_SERVER_WEBMAP_API=https://dev-k8s.treetracker.org/webmap/
-NEXT_PUBLIC_IMAGE_API=https://dev-k8s.treetracker.org/images
-NEXT_PUBLIC_API_NEW=http://127.0.0.1:4010/mock
-NEXT_PUBLIC_BASE=
-NEXT_PUBLIC_CONFIG_API=
-NEXT_PUBLIC_COUNTRY_LEADER_BOARD_DISABLED=false
-NEXT_PUBLIC_SERVER_CONFIG_DISABLED=true
```

Comment:
@RubenSmn this file is useful, we shouldn't delete it

Reply:
Yes I know, this PR is already closed :+1: @dadiorchen